Pipeline/routes/route.py

```diff
@@ -109,9 +109,23 @@ def get_graphs_per_module():
         abort(400, "Failed to retrieve representative graphs. Please check your inputs. Error: " + str(e))
 
 
-@routes.route('/rnamigos', methods=['GET'])
-def greeting_rna_migos():
-    return RnaMigos.greeting()
+@routes.route('/rnamigos_string', methods=['POST'])
```

Comment:
Endpoint not required for strings but you can leave it if you want

Reply:
I prefer leaving it for future features that might be added. I will modify the method comment from "Represents the RnaMigos endpoint for string input" to "For future use cases: represents the RnaMigos endpoint for string input.